agent/src/test/java/com/teamscale/jacoco/agent/options/AgentOptionsParserTest.java

```diff
@@ -0,0 +1,15 @@
+package com.teamscale.jacoco.agent.options;
+
+import com.teamscale.report.util.CommandLineLogger;
+import org.junit.Test;
+
+public class AgentOptionsParserTest {
```

Comment:
Missing interface comment

Reply:
interesting that TS doesn't complain about this. good catch!